### ISSUE TITLE
Context API is not supported?

### ISSUE THREAD

**Issue by matthew-dean** [NervJS/nerv#105]
I'm getting `React.createContext is not a function`. It would be nice to have an explicit list of what's not supported of the React API, instead of what is supported. Is there a nerv library to polyfill?

**Comment by matthew-dean** [comment]
Can I use something like this to polyfill the feature? https://github.com/valotas/preact-context

**Comment by dpikt** [comment]
I agree with your call for an explicit list of what’s *not* supported, it’s the first thing I look for with lightweight alternative libs like this.
